.pre-commit-config.yaml

```diff
@@ -20,18 +20,7 @@ repos:
   rev: 21.7b0
   hooks:
   - id: black
-    exclude: |
-      (?x)
-        ^src/pip/_internal/models|
-        ^src/pip/_internal/operations|
-        ^src/pip/_internal/\w+\.py$|
-        # Tests
-        ^tests/data|
-        ^tests/unit|
-        ^tests/functional/(?!test_install)|
-        ^tests/functional/test_install|
-        # A blank ignore, to avoid merge conflicts later.
-        ^$
+    exclude: src/pip/_vendor/
```

Comment:
```suggestion
```

I believe this line can be dropped as `_vendor` is already excluded globally on line 1.

Reply:
Indeed! Done. :)